osu.Game.Rulesets.Mania/Edit/Blueprints/ManiaSelectionBlueprint.cs

```diff
@@ -29,19 +31,13 @@ protected override void Update()
         {
             base.Update();
 
-            Position = Parent.ToLocalSpace(DrawableObject.ToScreenSpace(Vector2.Zero));
-        }
+            var anchor = scrollingInfo.Direction.Value == ScrollingDirection.Up ? Anchor.TopCentre : Anchor.BottomCentre;
+            Anchor = Origin = anchor;
+            foreach (var child in InternalChildren)
+                child.Anchor = child.Origin = anchor;
 
-        public override void Show()
-        {
-            DrawableObject.AlwaysAlive = true;
```

Comment:
Not sure why these are trimmed, but it seems like these were the only actual usages of this property. So it may be a good idea to remove them if they're no longer needed for some reason.

Reply:
It was unrelated to the changes of this PR, but the lifetime logic is no longer unnecessary (and conflicting) because `DrawableHitObject` has its own logic for that. I removed the lifetime logic.